src/site/filters/liquid.js

```diff
@@ -95,10 +95,12 @@ module.exports = function registerFilters() {
   liquid.filters.fileSize = data => `${(data / 1000000).toFixed(2)}MB`;
 
   liquid.filters.fileExt = data =>
-    data
-      .split('.')
-      .slice(-1)
-      .pop();
+    data && data.length
+      ? data
+          .split('.')
+          .slice(-1)
+          .pop()
+      : data;
```

Comment:
This seems symptomatic of a larger issue. If we're passing `null` or `undefined` to this function as a result of the transformers, we should figure out where that's happening and fix the transformer.

Reply:
I reverted the changes to `liquid.js` in this [commit](https://github.com/department-of-veterans-affairs/vets-website/pull/13962/commits/c0d4913c957908a116fbcc139c1ae40232306e5e). And addressed the issue in this [commit](https://github.com/department-of-veterans-affairs/vets-website/pull/13962/commits/b54f44358684a2ac6a25bead41bbfc026550a2e8). It was an issue with how the `media-image` and `media-document` transformers were setup.

Reply:
We will still need to work with the cms team to figure out the actual path that the `derivative` points to.